bugbug/models/regressor.py

```diff
@@ -128,8 +129,12 @@ def __init__(
             ]
         )
 
-        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
-        self.clf.set_params(predictor="cpu_predictor")
+        base_clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+        base_clf.set_params(predictor="cpu_predictor")
+        self.clf = IsotonicRegressionCalibrator(base_clf)
```

Comment:
Could you add a parameter to the constructor to enable/disable the calibration?

Reply:
@marco-c done!

Reply:
@WhiteWolf47 what you added isn't a parameter, but an attribute.
A parameter is a variable that is passed to the function (see for example `interpretable`).

Reply:
mb :sweat_smile: , i misunderstood it!